### PR TITLE
add DEVELOPMENT.md, who is using section

### PR DESCRIPTION
**Pull request by jiacai2050**
Remove redundant information and add who is using section

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -123,46 +114,24 @@ main(int argc, char** argv) {
...
+## Who is using
+VSAG has been integrated with the following database systems.
+
+<img alt="OceanBase" src="" height="60" />
+
+If your system use VSAG - feel free to make a PR to add it to the list.
+
+## Community
+Thrive together in VSAG community with users and developers from all around the world.
+- Discuss at [discord]()
```
Comment (jiacai2050) [inline README.md]: TBD

File: README.md [hunk 2]
```diff
@@ -123,46 +114,24 @@ main(int argc, char** argv) {
...
-`mockimpl/`: the mock implementation that can be used in interface test</br>
-`python_bindings/`: the python bindings</br>
-`scripts/`: useful scripts</br>
-`src/`: the source codes and unit tests</br>
-`tests/`: the functional tests</br>
+
+## Who is using
+VSAG has been integrated with the following database systems.
+
+<img alt="OceanBase" src="" height="60" />
```
Comment (jiacai2050) [inline README.md]: TBD

Comment (wxyucs) [reply README.md]: I'm collecting logos from partners. so maybe you can just remove this TBD section, I will edit it in the next commit

File: README.md (hunk 1 above)
Comment (wxyucs) [reply README.md]: the discord server URL is: https://discord.com/invite/JyDmUzuhrp

**Review by wxyucs — CHANGES_REQUESTED** [review]

**Comment by jiacai2050** [comment]
@wxyucs PTAL

**Comment by wxyucs** [comment]
@jiacai2050 I just updated the README file in the last commit, the REAMD.md file has a conflict now. Please mention me after you solve it, and then I will recheck this pull request.

**Comment by jiacai2050** [comment]
@wxyucs Conflict resolved.

**Review by wxyucs — APPROVED** [review]
lgtm

**Review by jiaweizone — APPROVED** [review]
LGTM

**Pull request merged**